bepdf/Makefile

```diff
@@ -128,7 +128,7 @@ OPTIMIZE := SOME
 # 	will recreate only the "locales/en.catkeys" file. Use it as a template
 # 	for creating catkeys for other languages. All localization files must be
 # 	placed in the "locales" subdirectory.
-LOCALES =
+LOCALES = en de
```

Comment:
Yes. Once you declare one locale in the Makefile you need to include the default "en" as well or it won't be compiled in.